timeless/reservations/views.py

```diff
@@ -115,15 +114,35 @@ def list_reservations(reservations):
 @bp.route("/create", methods=("GET", "POST"))
 def create():
     """ Create new reservation """
+    """
+    @todo #380:15min Continue. When we use the form.validate() function the
+     status (enum: timeless / restaurants / models.py) is displaying the
+     following error: 'status': ['Invalid Choice: could not coerce', 'Not a
+     valid choice']. Removes the commented if to see the error...
```

Comment:
@marceloamadeu Please add in this puzzle a reference to refactor the function using genericViews, as explained in  puzzles #134 and #173